### ISSUE TITLE
Swell client does not close connection

### ISSUE THREAD

**Issue by PatrickHeneise** [swellstores/swell-node#4]
I'm trying to test the integration with Swell and run into connection errors with jest:

```
 FAIL  test/swell.test.js (5.541 s)
  ✕ should return a list of orders (5044 ms)

  ● should return a list of orders

    : Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 5000 ms timeout specified by jest.setTimeout.Error:

      25 | })
      26 | 
    > 27 | test('should return a list of orders', async () => {
         | ^
      28 |   const response = await fastify.inject({
      29 |     method: 'GET',
      30 |     url: '/'

      at new Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:116:22)
      at Object.<anonymous> (test/swell.test.js:27:1)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        5.608 s, estimated 7 s
```

this is the file to reproduce the issue easily:

```
const swell = require('swell-node').init(
  process.env.SWELL_ID,
  process.env.SWELL_KEY,
  { cache: false }
)

const fastify = require('fastify')()

fastify.get('/', async function (request, reply) {
  const orders = await swell.get('/orders', {
    limit: 10,
    page: 1
  })
  reply.send(orders)
})

fastify.listen(3000, function (err, address) {
  if (err) {
    fastify.log.error(err)
    process.exit(1)
  }
  fastify.log.info(`server listening on ${address}`)
})

test('should return a list of orders', async () => {
  const response = await fastify.inject({
    method: 'GET',
    url: '/'
  })
  expect(response.statusCode).toEqual(200)
})
```

**Comment by ericingram** [comment]
@PatrickHeneise I think you got an answer on this in slack, did that work? (to increase the async timeout)

**Comment by PatrickHeneise** [comment]
Unfortunately not, still getting the same, with fairly long timeouts: `list.test.js (16.525 s)`

Also tried >60s, nothing. I added `swell.close()` before the reply, doesn't help either.

**Comment by ericingram** [comment]
Sounds like the way the test is written or something, it shouldn't expect
the connection to close... is the request returning successfully?

**Comment by PatrickHeneise** [comment]
No, it's stuck. I'll try again today

**Comment by MoSattler** [comment]
@ericingram I have the same problem. Bluebird does not seem to work well with jest. I had to manually update bluebird to get it to work.

**Comment by ericingram** [comment]
@MoSattler good catch - fixed and published in v3.1.6
